pytensor/link/numba/dispatch/random.py

```diff
@@ -386,50 +386,45 @@ def numba_funcify_RandomVariable_core(op: RandomVariable, **kwargs):
     )
 
 
-@numba_funcify.register
-def numba_funcify_RandomVariable(op: RandomVariableWithCoreShape, node, **kwargs):
-    core_shape = node.inputs[0]
+def rv_fallback_impl(op: RandomVariableWithCoreShape, node):
+    """Create a fallback implementation for random variables using object mode."""
+    import warnings
 
     [rv_node] = op.fgraph.apply_nodes
     rv_op: RandomVariable = rv_node.op
+
+    warnings.warn(
```

Comment:
We already have a generic fallback implementation function can't we just use it like we do for other Ops?

May just need to do the unboxing of the RV that the other function is doing